test/types/enum/lydia/use/useMultiple.chpl

```diff
@@ -5,9 +5,8 @@ module M {
 
   enum bar {d, e, f};
 
-  use foo, bar;
-
   proc main() {
+    use foo, bar;
```

Comment:
moved to inner scope to avoid conflict between `bar.e` and (automatically included) `Math.e`.

Reply:
Huh, I hadn't thought of this before now, but it seems odd to me that a [private] `use` of an enum would put the enum's symbols into a shadow scope in the case that lives in the enum's scope itself.  I.e., `bar.e` seems obviously much closer than `Math.e` to this file's module, by virtue of it being declared right here in this scope, and I think of the `use` as being "let me just bypass the fully-qualified naming" rather than "please insert shadow scopes so that I can refer to these symbols."

Would we just want to say that `use` of enums never creates a shadow scope?  That would mean that if the enum was in a different module and I `use`d it in my client module, I'd have an ambiguity if I had a local variable named d, e, or f, which also seems attractive (rather than my local variable winning over the enum)—or at least, not unattractive (?).

Cases like this, and the `round()` case make me a bit sad given that I expect those symbols to end up in a non-auto-used module before too long.  Whatever we do w.r.t. my preceding few paragraphs, I might be inclined to file a clone of this as a future that will "work" once `Math.e` is no longer auto-available (or just futurize this one until that day occurs).

Reply:
One thing that just occurred to me - I thought the use of ChapelStandard we insert by default was private.  If that's true, it seems like Math.e should be at a further scope than anything brought in by a public use and anything defined in the module, right?  Is the publicness of ChapelStandard's use of Math bleeding through when it shouldn't?

Reply:
If we have this:

``` chapel
module M {
  enum foo {a, b, c};

  enum bar {d, e, f};

  use foo, bar;

  proc main() {
    writeln(a, " ", b, " ", c);
    writeln(d, " ", e, " ", f);
  }
}
```

then the `use foo, bar` creates a shadow scope for a/b/c/d/e/f.
There is also an implicit `use ChapelStandard` that creates a shadow scope that includes `e` (from `Math.e`).

As a result both `bar.e` and `Math.e` end up in the shadow scope that is created to store the symbols for these private `use` statements.

Reply:
Ah, right!  Okay, never mind then

Reply:
I've created
 * #19367

to discuss the use-enum-does-not-create-shadow-scope proposal further

Reply:
Vaguely related, I added some comments to the OP of https://github.com/chapel-lang/chapel/issues/18826 on Friday about how public/private would interplay with the feature proposed there.